App crashes when no WebView installed

**Describe the bug**

https://github.com/jellyfin/jellyfin-android/blob/1431e59cfc4fca0a6c55c3f425e4eabcff921ec1/app/src/main/java/org/jellyfin/mobile/JellyfinApplication.kt#L27

^ This check can crash the app if there is no webview. Should probably use the `isWebViewSupported()` function from WebViewUtils.

**To Reproduce**
<!-- Steps to reproduce the behavior: -->
1. disable webview app
2. ??
3. Profit

**Expected behavior**
No crash

**Logs**
```php
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.jellyfin.mobile.debug, PID: 13606
    java.lang.RuntimeException: Unable to create application org.jellyfin.mobile.JellyfinApplication: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6717)
        at android.app.ActivityThread.access$1300(ActivityThread.java:237)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: android.util.AndroidRuntimeException: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
        at android.webkit.WebViewFactory.getProviderClass(WebViewFactory.java:435)
        at android.webkit.WebViewFactory.getProvider(WebViewFactory.java:252)
        at android.webkit.WebView.getFactory(WebView.java:2576)
        at android.webkit.WebView.setWebContentsDebuggingEnabled(WebView.java:1999)
        at org.jellyfin.mobile.JellyfinApplication.onCreate(JellyfinApplication.kt:27)
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1192)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6712)
        at android.app.ActivityThread.access$1300(ActivityThread.java:237) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7656) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
     Caused by: android.webkit.WebViewFactory$MissingWebViewPackageException: Failed to load WebView provider: No WebView installed
        at android.webkit.WebViewFactory.getWebViewContextAndSetProvider(WebViewFactory.java:339)
        at android.webkit.WebViewFactory.getProviderClass(WebViewFactory.java:402)
        at android.webkit.WebViewFactory.getProvider(WebViewFactory.java:252) 
        at android.webkit.WebView.getFactory(WebView.java:2576) 
        at android.webkit.WebView.setWebContentsDebuggingEnabled(WebView.java:1999) 
        at org.jellyfin.mobile.JellyfinApplication.onCreate(JellyfinApplication.kt:27) 
        at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1192) 
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6712) 
        at android.app.ActivityThread.access$1300(ActivityThread.java:237) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:223) 
        at android.app.ActivityThread.main(ActivityThread.java:7656) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947) 
```

**Screenshots**
![image](https://user-images.githubusercontent.com/2305178/115067992-efa7f180-9ef1-11eb-8ec3-1fe1a1e6ed75.png)

**System (please complete the following information):**
 - App version: apiclient-1.0 branch
 - Android version: Emulator API 30
 - Device manufacturer and model: Google Emulator
 - Jellyfin server version: 11.0.0
 - The type of video player implementation used: Windows Media Player

**Additional context**
Probably could've fixed it instead of writing this issue.

--- a/app/src/main/java/org/jellyfin/mobile/JellyfinApplication.kt
+++ b/app/src/main/java/org/jellyfin/mobile/JellyfinApplication.kt
@@ -6,6 +6,7 @@ import com.melegy.redscreenofdeath.RedScreenOfDeath
 import org.jellyfin.mobile.api.apiModule
 import org.jellyfin.mobile.model.databaseModule
 import org.jellyfin.mobile.utils.JellyTree
+import org.jellyfin.mobile.utils.isWebViewSupported
 import org.koin.android.ext.koin.androidContext
 import org.koin.androidx.fragment.koin.fragmentFactory
 import org.koin.core.context.startKoin
@@ -24,7 +25,9 @@ class JellyfinApplication : Application() {
             RedScreenOfDeath.init(this)
 
             // Enable WebView debugging
-            WebView.setWebContentsDebuggingEnabled(true)
+            if (isWebViewSupported()) {
+                WebView.setWebContentsDebuggingEnabled(true)
+            }
         }
 
         startKoin {

